content/es/community/marketing-guidelines.md

```diff
@@ -0,0 +1,114 @@
+
+### I: OpenTelemetry es un esfuerzo conjunto {#i-open-telemetry-is-a-joint-effort}
+
+- Consejos:
+  - Usar materiales del proyecto, como el logotipo y el nombre, en consonancia
+    con la imagen de marca de la Fundación Linux y
+    [las directrices de uso de la marca registrada](https://www.linuxfoundation.org/legal/trademark-usage)
+  - Enfatizar que OTel no sería posible sin la colaboración de muchos
+    colaboradores que trabajan para proveedores de la competencia
+  - Citar los nombres de otros colaboradores y proveedores involucrados en las
+    iniciativas de OTel
+  - Enfatizar nuestros objetivos comunes como comunidad para mejorar las
+    experiencias de los usuarios finales/desarrolladores y empoderarle
+- Qué no hacer:
+  - Dar a entender que un único proveedor es responsable de OTel o de alguno de
+    sus componentes.
+  - Menospreciar las contribuciones de otra organización o persona.
+
+### II: No es una competición {#ii-its-not-a-competition}
+
+- Consejos:
+  - Enfatizar que todas las contribuciones son valiosas y que son de diversas
+    formas y tamaños, incluyendo:
+  - Contribuciones al código principal del proyecto o a SDK específicos del
+    lenguaje o framework
+  - Crear y compartir recursos educativos (videos, talleres, artículos) o
+    recursos compartidos que puedan usarse con fines educativos (por ejemplo,
+    una aplicación de muestra que use un lenguaje o framework específico)
+  - Actividades para fomentar la comunidad, como organizar un evento o una
+    reunión
+  - Reconocer y agradecer públicamente a otras organizaciones por sus
+    contribuciones a OTel
+- Qué no hacer:
+  - Comparar directamente el volumen o el valor de los diferentes contribuyentes
+    a OTel (por ejemplo, a través de [CNCF devstats](https://devstats.cncf.io/))
+  - Insinuar que quienes contribuyen poco o con poca frecuencia a OTel son
+    necesariamente ciudadanos de segunda clase, y/o que su propia compatibilidad
+    con OTel debería cuestionarse como resultado (de hecho, no hay razón para
+    que un proveedor deba contribuir a OTel para apoyarlo)
+
+### III: Promover la concienciación sobre la interoperabilidad y modularización de OTel {#iii-promote-awareness-of-otel-interoperability-and-modularization}
+
+- Consejos:
+  - Dar a conocer la compatibilidad con OTel: cuanto mejor comprendan los
+    usuarios finales lo que pueden hacer con los datos de OTel, mejor.
+  - Enfatizar la neutralidad del proveedor y la portabilidad de cualquier
+    integración con OTel.
+- Qué no hacer:
+  - Insinuar que un usuario final no está "usando OTel" a menos que utilice un
+    conjunto específico de componentes dentro de OTel (OTel es un proyecto
+    "amplio" con muchos componentes desacoplados).
+  - Denigrar públicamente el soporte de OTel de otro proveedor, especialmente
+    sin pruebas objetivas.
+
+## Preocupaciones y consecuencias {#concerns-and-consequences}
```

Comment:
Maybe `Inquietudes` instead of `Precupaciones`, sounds more coherent with the text below `gestionar estas inquietudes`